# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  3d34236b27abcc0d05c4a71edf662d302e9cc371
Branding: Replace MOZ_APP_NAME in nsGNOMEShellService.cpp.

diff --git a/mail/components/shell/nsGNOMEShellService.cpp b/mail/components/shell/nsGNOMEShellService.cpp
--- a/mail/components/shell/nsGNOMEShellService.cpp
+++ b/mail/components/shell/nsGNOMEShellService.cpp
@@ -91,9 +91,9 @@ nsresult nsGNOMEShellService::Init() {
   rv = NS_GetSpecialDirectory(NS_XPCOM_CURRENT_PROCESS_DIR,
                               getter_AddRefs(appPath));
   NS_ENSURE_SUCCESS(rv, rv);
 
-  rv = appPath->AppendNative(nsLiteralCString(MOZ_APP_NAME));
+  rv = appPath->AppendNative(nsLiteralCString("betterbird"));
   NS_ENSURE_SUCCESS(rv, rv);
 
   rv = appPath->GetNativePath(mAppPath);
   return rv;
